rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -436,3 +436,30 @@ def _list_oflow_count_for_sandboxes(self, sandboxes,
             "chart_plugin": "StackedArea", "data": oflow_data
         }
         self.add_output(additive_oflow_data)
+
+    def _create_address_set(self, set_name, address_list):
+        LOG.info("create %s address_set [%s]" % (set_name, address_list))
+
+        name = "name=\"" + set_name + "\""
+        addr_list="\"" + address_list + "\""
+
+        ovn_nbctl = self.controller_client("ovn-nbctl")
+        ovn_nbctl.set_sandbox("controller-sandbox", self.install_method)
+        ovn_nbctl.create("Address_Set", name, ('addresses', addr_list))
```

Comment:
I am awfully sorry that after taking a second look, it seems we still need a flush() here, since I noticed in many places the existing code just enabled batch mode without restoring it back to non-batch mode after flush(). It deserves a separate patch to fix those issues (probably changing flush() to restore to non-batch mode as default behavior). But here I think it is good enough to just flush() after the operation. If it is not in batch mode, the flush() is no-op. Would you mind revise again?

Reply:
No worries, do you mean to add just a flush() after the operation or also enable the batch before db access?

Reply:
I meant just flush().